.github/workflows/build-and-publish.yaml

```diff
@@ -0,0 +1,45 @@
+name: Build and Publish Assets
+on: 
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Version to be released'
+        required: true
+        default: 'stale'
```

Comment:
Stale or "stable"?

Reply:
version is mandatory, so stale is like an invalid version

Reply:
made it stable now.